testdata/p4tc_samples_outputs/default_action_with_param_01.template

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -x
+
+set -e
+
+TC="tc"
+$TC p4template create pipeline/default_action_with_param_01 pipeid 1 numtables 2
+
+$TC p4template create action/default_action_with_param_01/MainControlImpl/next_hop actid 1
+$TC p4template update action/default_action_with_param_01/MainControlImpl/next_hop state active
+
+$TC p4template create action/default_action_with_param_01/MainControlImpl/dflt_route_drop actid 2
+$TC p4template update action/default_action_with_param_01/MainControlImpl/dflt_route_drop state active
+
+$TC p4template create action/default_action_with_param_01/MainControlImpl/drop actid 3
+$TC p4template update action/default_action_with_param_01/MainControlImpl/drop state active
+
+$TC p4template create table/default_action_with_param_01/MainControlImpl/ipv4_tbl_1 \
+	tblid 1 \
+	type exact \
+	keysz 64 nummasks 8 tentries 2048 \
+	table_acts act name default_action_with_param_01/MainControlImpl/next_hop \
```

Comment:
I don't know whether this is related to this PR or not, but the `defaultonly` flag is not being assigned for `default_action_with_param_01/MainControlImpl/next_hop` in the template, even though you are putting it in the P4 program